README.md

```diff
@@ -209,11 +173,11 @@ The last number on each line is the time (in seconds) current operation takes. Y
 - Local disk
 - Redis
 
-For the detailed list, see [this document](docs/en/how_to_setup_object_storage.md#supported-object-storage).
+JuiceFS supports almost all object storage services. [Learn more](docs/en/how_to_setup_object_storage.md#supported-object-storage).
 
 ## Status
 
-It's considered as beta quality, the storage format is not stabilized yet. It's not recommended to deploy it into production environment. Please test it with your use cases and give us feedback.
+It's considered as beta quality, the storage format is not stabilized yet. It's not recommended to deploy it into production environment. Please test it with your use cases and give us [feedback](https://github.com/juicedata/juicefs/issues).
```

Comment:
Maybe link to GitHub Discussions? GitHub Issues is for feature request and bug report.